### PR TITLE
run cargo fmt

### PR DESCRIPTION
**Pull request by gdesmott**
Fix coding styles.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -503,7 +511,7 @@ impl Future for BackgroundTask {
             match maybe_maybe_item {
                 Some(Some(item)) => self.queues[item.level].push(item),
                 Some(None) => self.quitting = true, // Explicit close.
-                None => self.quitting = true, // The sender was dropped.
+                None => self.quitting = true,       // The sender was dropped.
```
Comment (hrxi) [inline src/lib.rs:514]: Questionable choice. I couldn't find a [rustfmt option](https://rust-lang.github.io/rustfmt/) that disables it though.

**Review by hrxi — CHANGES_REQUESTED** [review]
Looks generally good to me. Needs a Github Actions CI check that the formatted code is actually cargo-fmt'ed, otherwise it'll get out of sync again.

File: src/labels.rs [hunk 2]
```diff
@@ -78,11 +78,26 @@ mod test {
...
+            FormattedLabels::new().finish(Level::INFO),
+            r#"{level="info"}"#
+        );
+        assert_eq!(
+            FormattedLabels::new().finish(Level::WARN),
+            r#"{level="warn"}"#
+        );
+        assert_eq!(
+            FormattedLabels::new().finish(Level::ERROR),
+            r#"{level="error"}"#
```
[the remark this answers is not in the document]
Comment (gdesmott) [reply src/labels.rs]: Added those.

**Comment by gdesmott** [comment]
> Needs a Github Actions CI check that the formatted code is actually cargo-fmt'ed, otherwise it'll get out of sync again.

I configured my editor (`vscode`) to automatically run `rustfmt` whenever a file is written, so I don't have to ever bother about this. It's quite handy.

**Comment by hrxi** [comment]
> I configured my editor (`vscode`) to automatically run `rustfmt` whenever a file is written, so I don't have to ever bother about this. It's quite handy.

That solves the problem for you. I think this doesn't help on a shared project though. I won't be able to verify that a PR is cargo-fmt'ed by looking at it.

**Comment by gdesmott** [comment]
> I think this doesn't help on a shared project though. I won't be able to verify that a PR is cargo-fmt'ed by looking at it.

Sure, I have a CI job checking formatting in all my projects. I'm more of a Gitlab person, but I think I may have something in `system-dep`'s CI I could re-use.

**Comment by gdesmott** [comment]
Oh you already have some CI, then it's easy. Should be good but I can't run the CI pipeline myself. :\

**Comment by gdesmott** [comment]
Looks like `rustfmt` is not installed in the nightly image. Should we run it only on stable then?

**Comment by hrxi** [comment]
> Looks like `rustfmt` is not installed in the nightly image. Should we run it only on stable then?

That would work, or you could install rustfmt as well: https://github.com/actions-rs/toolchain/tree/88dc2356392166efad76775c878094f4e83ff746#components. Both are fine.

**Comment by gdesmott** [comment]
I disabled it as I suppose nightly could require a different format because of changes in `rustfmt`.

**Review by hrxi — APPROVED** [review]
Thanks!

**Pull request merged**